DEVELOP.md

```diff
@@ -230,4 +230,21 @@ taco file generated in the release.
 3. Create a new tag following the convention with the Target branch `main`.
 4. Click on `Generate release notes`, amend the notes as needed and publish.
 5. The pipeline will run and upload the taco file to the releases' assets, check that the pipeline
-runs correctly and the file exists.
\ No newline at end of file
+runs correctly and the file exists.
+
+
+## Signature verification
+
+If you want to install a release that is not signed, signature verification can
+be disabled.
+
+* On _Tableau Desktop_, please use this command-line argument:
+  `-DDisableVerifyConnectorPluginSignature=true`
+
+* On _Tableau Server_, you can disable signature verification by setting
+  `native_api.disable_verify_connector_plugin_signature` to `true` via TSM.
+
+More information can be found on the [Tableau Connector SDK `run-taco` documentation].
+
+
+[Tableau Connector SDK `run-taco` documentation]: https://tableau.github.io/connector-plugin-sdk/docs/run-taco
```

Comment:
FYI: I didn't want to delete this section, so I just relocated it into `DEVELOP.md`. Do you agree?

Reply:
I would just delete it. It is not needed while developing since one would typically run the connector as is, in the repo, without building.